tests/acceptance/conftest.py

```diff
@@ -109,6 +110,20 @@ def project(gl: Gitlab, group: Group):
     gitlab_project.delete()
 
 
+@pytest.fixture(scope="class")
+def project_hook(gl: Gitlab, project):
+    name = get_random_name("hook")
+    hook = project.hooks.create({"url": f"http://hooks/{name}.org"})
+
+    yield hook
+    # When using in hook deletion test this is Not Found
+    try:
+        hook.delete()
+    except (GitlabDeleteError, GitlabHttpError) as e:
+        if e.response_code == 404:
+            pass
+
```

Comment:
Would it be better to move this fixture into the `tests/acceptance/standard/test_hooks_processor.py` file? Current pattern is that this file contains fixtures that are used across different feature tests. Do you see this fixture being used in other feature tests?

Reply:
I wasn't sure, I thought that maybe it didn't matter, for tidiness' sake. I'll move it - thanks for all the patience, it's really my only way to learn on here!

Reply:
No worries. I'm learning too. 😄